web_src/css/actions.css

```diff
@@ -57,3 +57,25 @@
   gap: .25rem;
   align-items: center;
 }
+
+.run-list .flex-item-trailing {
+  flex-wrap: nowrap;
+}
+
+.run-list-ref {
+  display: inline-block !important;
+  max-width: 200px;
+}
+
+@media (max-width: 767.98px) {
+  .run-list .flex-item-trailing {
+    flex-direction: column;
+    align-items: flex-end;
+  }
+  .run-list-item-right {
+    width: 90px;
```

Comment:
I just feel curious, does this hard-coded width fits all languages? On the screenshot I can see "Last week", it fits, but the text could be very long in other languages.

Reply:
Likely not but worst case it will start to wrap which is acceptable there imho.

Reply:
I would reluctantly accept the hard-coded width if it is the only solution. If it is possible to avoid the hard-coding and make the UI layout by the contents, it would look better to me.

Reply:
Should be possible to layout by content but long branch name will use a lot of space without this and I think branch name is not really that interesting for a mobile viewer. So likely could just limit that branch name there to this 90px and the column will then downsize.

Reply:
Sorry but I do not understand how the "run-list-item-right / 90px" is related to "branch name":

```
				<div class="run-list-item-right">
					<div class="run-list-meta">{{svg "octicon-calendar" 16}}{{TimeSinceUnix .Updated ctx.Locale}}</div>
					<div class="run-list-meta">{{svg "octicon-stopwatch" 16}}{{.Duration}}</div>
				</div>
```

I didn't see "branch name" in code ........

Reply:
Ah yes the branch label is not part of it, but it does shift into that "column" on mobile layout as seen in screenshot above, this is intentional:

<img width="160" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/f62d402f-6dd3-4ebc-9f92-2943d910eb31">

Reply:
But what if in some languages the "last week" is much longer? For example, Russia?

Reply:
Once the limit is moved to the branch tag, it will stretch it the time.

Reply:
Now it's 100px and effective on tag and right side:

<img width="440" alt="Screenshot 2024-04-13 at 21 43 46" src="https://github.com/go-gitea/gitea/assets/115237/8e5ee45b-9136-4782-9699-2a3e7f363755">

Reply:
What if the "time" text has something longer? eg: на прошлой неделе (Although I am not sure it is the correct translation, I am indeed sure that Russian is much longer than many languages)

Reply:
As I said, it will wrap the time which is acceptable imho.